l2munger.cpp

```diff
@@ -247,17 +247,21 @@ time_t targetVolumeTime;
 time_t timeDelta;
 
 time_t volumeTime;
+int    speedFactor;
 
 
 //******************************************************************************
 
 void AdjustW88Time( uint16 &w88_days, uint32 &w88_seconds )
  {
-  time_t seconds;  
+  time_t seconds;
+  time_t innerTimeDelta;  
 
   seconds = W88TimeToTime( w88_days, w88_seconds );
 
-  seconds += timeDelta;
+  innerTimeDelta = seconds - volumeTime;
+
+  seconds += timeDelta - innerTimeDelta + (innerTimeDelta / speedFactor);
```

Comment:
Here is where the value is used...it just makes the time between the volume time and the product time be `speedFactor` times faster.

Reply:
strange, glossed over that.